Rakefile

```diff
@@ -34,3 +34,14 @@ task :seed_catalog, [:profile] => [:environment] do |_t, args|
   puts "#{Time.now.utc.iso8601} Done"
   $stdout.flush
 end
+
+
+task :drop, [:storage_root] => [:environment] do |t,args|
+  if args[:storage_root]
+    p "you're in the if condition"
```

Comment:
It would be awesome if this rake task had a "you're about to erase all data for xxx - are you sure? (Y, N)" sort of prompt.  

This would be more complex than your puts statement above (which I assume is debug speak), but I think it's the right thing to do.